grist-doc-wk-exporter/build-push.sh

```diff
@@ -0,0 +1 @@
+docker build -t ghcr.io/betagouv/grist-utils/grist-doc-wk-exporter:latest --push .
```

Comment:
Could we make the container registry an env variable maybe ? If we want to have other people use it ?